src/Components/Grid/Grid.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Character from '../Character/Character';
+import {Link} from 'react-router-dom';
+import './Grid.css';
+
+const Grid = ({characters, findPlayer, addPlayer}) => {
+
+
+  const players = characters.map(character => {
+
+    const player = character[0];
+    const firstName = player.name.split(' ')[0];
+
+    return (
+      <div>
+        <Link to={`/${player.name.split(' ')[0].toLowerCase()}`} style={{textDecoration: 'none'}}>
```

Comment:
Is there a specific reason for using a style property here rather than doing the text decoration in css?